src/csharp/Microsoft.Spark.UnitTest/Sql/TimestampTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Sql.Types;
+using Xunit;
+
+namespace Microsoft.Spark.UnitTest
+{
+    public class TimestampTests
+    {
+        [Fact]
+        public void TimestampTest()
+        {
+            {
+                var timestamp = new Timestamp(
+                    new DateTime(2020, 1, 1, 8, 30, 30, DateTimeKind.Utc).AddTicks(1230));
```

Comment:
Shall we create `var testDate = new DateTime(2020, 1, 1, 8, 30, 30, DateTimeKind.Utc).AddTicks(1230);` and reuse it below?

Reply:
Sounds good. Changed!